articles/2016-09-01-semantic-html-tags.md

```diff
@@ -0,0 +1,88 @@
+# :name_badge: Semantic HTML Elements
+
+For this article, I assume you have a basic working knowledge of HTML. If this is not the case, please [email me](mailto:maddie.horowitz@addepar.com) and I will bring you up to speed.
+
+### HTML5
+
+HTML5 was the newest iteration of of HTML published in 2014. This version primarily focused on enabling multiple devices and systems to interpret the markup languge. With a growing focus on responsive websites, screen readers and accessibility, the WWW introduced semantic tags. Specifically, they introduced new tags to represent new formats of media (`<canvas>`, `<video>`, `<audio>`), as well as ones to represent page structure (`<main>`, `<section>`, `<header>`, `<footer>`, `<nav>`, etc). For a full list of all available elements offered as of HTML 5, please refer to the [documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element).
+
+### Why are semantic tags better?
+
+The goal of semantic HTML elements is to better represent the content within. A `<div>` can represent just about any block-level element, but it is better to be more specific. Screen readers, parsers, and search engines that try to read your HTML and attempt to understand its structure. If you use semantic tags. you are giving these systems an easier time.
```

Comment:
`If you use semantic tags. you are giving these systems an easier time.` needs a comma instead of period in the middle